### DISCUSSION TITLE
Make a Video Showing Features for First-Time Users

### DISCUSSION THREAD

**Discussion by rmmays** [rxhanson/RectanglePro-Community#510] [Ideas]
Ryan, i love your app, but could i get a video showing how to use it most effectively?  Maybe just the basic 'hello world' then a few "tips and tricks"?

**Comment by rxhanson** [comment]
Here's a short video that I made on the Window Throw functionality:

https://www.youtube.com/watch?v=Vo09H-ln7BQ

Here's another discussion about this: https://github.com/rxhanson/RectanglePro-Community/discussions/305, with a few user-created videos, too.

I gotta get back into it and make some more.
